### PR TITLE
Add AutoFillFormSuggestion for select,textarea in form

### PR DESCRIPTION
**Pull request by alban-stourbe-wmx**
I tried to add the detection of textearea and select in a form, respecting what was already implemented. This is the first time I've worked on a GO project. I don't know whether I'm using best practices and whether my proposal is as modular as possible. Don't hesitate to point out any pieces of code that seem badly written. 

PR based on the issue #456

### CODE REVIEW

**Comment by GeorginaReeder** [comment]
Thanks so much for your contribution @alban-stourbe-wmx !

File: pkg/utils/formfill.go [hunk 1]
```diff
@@ -106,6 +125,78 @@ func FormInputFillSuggestions(inputs []FormInput) mapsutil.OrderedMap[string, st
...
+//
+// Returns:
+// An ordered map containing suggestions for filling the form fields.
+func FormFillSuggestions(formFields []interface{}) mapsutil.OrderedMap[string, string] {
+	merged := mapsutil.NewOrderedMap[string, string]()
+	for _, item := range formFields {
+		switch v := item.(type) {
+		case FormInput:
+			dataMapInputs := FormInputFillSuggestions([]FormInput{v})
+			dataMapInputs.Iterate(func(key, value string) bool {
```
Comment (dogancanbakir) [inline pkg/utils/formfill.go:179]: almost identical to other instances, this can be a small func: func Merge(...) {...}

**Review by dogancanbakir — APPROVED** [review]

File: pkg/utils/formfill.go [hunk 2]
```diff
@@ -41,6 +41,25 @@ type FormInput struct {
 	Attributes mapsutil.OrderedMap[string, string]
 }
 
+// FormOption is an option for a select input
+type FormOption struct {
```
[the remark this answers is not in the document]
Comment (alban-stourbe-wmx) [reply pkg/utils/formfill.go]: It is fixed

File: pkg/utils/formfill.go (hunk 1 above)
Comment (alban-stourbe-wmx) [reply pkg/utils/formfill.go]: You are right, i made a small function to merge OrderedMaps

**Comment by alban-stourbe-wmx** [comment]
@Mzack9999 @dogancanbakir I have changed what you were asking. I hope, it is good for you now

**Review by Mzack9999 — APPROVED** [review]

**Comment by Mzack9999** [comment]
Closes #456

**Pull request merged**